examples/geospatial/plot_delaunay.py

```diff
@@ -0,0 +1,54 @@
+"""
+=====
+Graphs from geographic points
```

Comment:
There are two examples [in the gallery](https://807-890377-gh.circle-artifacts.com/0/doc/build/html/auto_examples/index.html#geospatial) with the same title. It seems like this example could be named differently, maybe "Delaunay graph from geometric points" or similar?

Reply:
Yes, I've made this more specific in the new text.